hawkular-component/src/main/java/org/hawkular/metrics/security/InfluxAuthHttpHandler.java

```diff
@@ -41,6 +40,12 @@
  * @author Thomas Segismont
  */
 public class InfluxAuthHttpHandler implements HttpHandler {
+    private static final Pattern UUID_PATTERN;
+
+    static {
```

Comment:
Just out of curiosity: why is this wrapped into a static block? You have only one statement and there's no exception to catch, so, it has the same effect as if you were assigning directly at the declaration above, no ?